projects/ltc7841/README.rst

```diff
@@ -0,0 +1,128 @@
+Evaluating the LTC7841
+======================
+
+.. contents::
+	:depth: 3
+
+Supported Evaluation Boards
+---------------------------
+
+* `DC2798A <https://www.analog.com/en/resources/evaluation-hardware-and-software/evaluation-boards-kits/DC2798A.html>`_
+
+Overview
+--------
+
+The evaluation board includes 2 LTC7841 to be powered up with writing to the MFR_VOUT_COMMAND a non-zero value.
+To turn-on the evaluation board, write 0x80 to the OPERATION register and ensure that the MFR_VOUT_COMMAND is actually
+non-zero value.
+
+Full performance details are provided in the LTC7841 data sheet, which should
+be consulted in conjunction with user guide.
+
+Hardware Specifications
+-----------------------
+
+Power Supply Requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For this specific project an external power supply with a variable output of 10-16 Volts
+must be used to power up the demo board.
+
+**Hardware Bringup**
+
+For reference, follow the Quick Start Procedure section of the corresponding
+demo board.
+`user guide <https://www.analog.com/media/en/technical-documentation/user-guides/dc2798a.pdf>`_.
+
+No-OS Build Setup
+-----------------
+
+Please see: https://wiki.analog.com/resources/no-os/build
+
+No-OS Supported Examples
+------------------------
+
+The initialization data used in the examples is taken out from:
+`Project Common Data Path <https://github.com/analogdevicesinc/no-OS/tree/main/projects/ltc7841/src/common>`_
+
+The macros used in Common Data are defined in platform specific files found in:
+`Project Platform Configuration Path <https://github.com/analogdevicesinc/no-OS/tree/main/projects/ltc7841/src/platform>`_
+
+Basic example
+^^^^^^^^^^^^^
+
+This is a simple example that initializes the LTC7841, and performs telemetry
+readings of the voltage, current and temperature of each output channel. Status
+bytes/words are also monitored in the example.
+
+In order to build the basic example make sure you have the following configuration in the Makefile
+`Makefile <https://github.com/analogdevicesinc/no-OS/tree/main/projects/ltc7841/Makefile>`_
+
+.. code-block:: bash
+
+	# Select the example you want to enable by choosing y for enabling and n for disabling
+	BASIC_EXAMPLE = y
```

Comment:
i don't think editing Makefiles is good to set the example to build even though we have existing projects done this way... let's strive to make the user interaction easy so that they run one of these commands to choose the example to build:

```
make EXAMPLE=basic_example
make EXAMPLE=iio_example
```

Basically you could default to basic like this:
```
EXAMPLE ?= basic_example
```

And take action based on it:
```
ifeq ($(INTERFACE), usb)
...
endif
```
```